packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -284,4 +288,20 @@ Will double processing times. Use only for debugging purposes.",
     default: defaultOptions.chain.maxCPStateEpochsInMemory,
     group: "chain",
   },
+
+  "chain.targetBlobsPerBlock": {
```

Comment:
these values will be part of spec config

Reply:
Setting 4/8 or 6/9 is part of the scope of EIP-7691. Will open a separate PR for it.

EIP-7742 only sets target = max / 2